examples/scripts/iframe/index.mustache

```diff
@@ -117,17 +92,17 @@
                 }).observe(canvasContainerElement);
             }
 
-            return app;
-        }
-
-        function hasBasisAssets (assets) {
-        for (let i = 0; i < assets.length; i++) {
-            if (assets[i].url && assets[i].url.includes('.basis')) {
-                return true;
+            if (location.pathname.indexOf('misc/mini-stats') === -1) {
```

Comment:
could we add another variable to the example to control this instead of here? We already have NAME and CATEGORY and few more. For example 
`static MINISTATS = false;`